content/syllabuses/data-sci.md

```diff
@@ -9,29 +9,29 @@ _db_id: 2
 - {{% contentlink path="topics/solo-learn/python/module-5-more-types" %}}
 - {{% contentlink path="projects/katas/level-2" flavour="python" %}}
 - {{% contentlink path="language-agnostic/intro-to-hackerrank" flavour="python" %}}
-- {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
 - {{% contentlink path="workshops/general-coding-best-practices" %}}
-- {{% contentlink path="projects/data-science-specific/statistical-thinking" flavour="python" %}}
 - {{% contentlink path="topics/python-specific/oop-for-python" %}}
 - {{% contentlink path="topics/python-specific/automated-testing-with-pytest"%}}
-- {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
 - {{% contentlink path="topics/python-specific/virtual-envs/" %}}
+- {{% contentlink path="projects/data-science-specific/data-wrangling" flavour="python" %}}
+- {{% contentlink path="projects/data-science-specific/statistical-thinking" flavour="python" %}}
+- {{% contentlink path="projects/data-science-specific/data-visualisation" %}}
 - {{% contentlink path="topics/survey-design" %}}
 - {{% contentlink path="topics/google-forms-like-a-boss" %}}
 - {{% contentlink path="projects/data-science-specific/cross-validation-and-simple-linear-regression/" flavour="python" %}}
 - {{% contentlink path="projects/morse-code" flavour="python" %}}
 - {{% contentlink path="workshops/intro-to-data-ethics" %}}
+- {{% contentlink path="topics/data-ethics-and-privacy" %}}
 - {{% contentlink path="projects/tdd/string-calculator" flavour="python" %}}
 - {{% contentlink path="projects/data-science-specific/multivariate-linear-regression" flavour="python" %}}
 - {{% contentlink path="projects/github-api-consume" flavour="python" %}}
 - {{% contentlink path="projects/oop/animals/part2"  flavour="python" %}}
```

Comment:
how about just animals part 1 as a little intro to oop. Possibly add animals part 2 way at the end, just in case people run out of stuff to do?

Reply:
I don't think people running out of stuff to do is an issue, for most learners it's getting through everything that's the problem. Also, animals part 2 isn't really about OOP since its just adding tests to animals part 1. If we leave it out then we'll have three OOP projects, I think that should be enough:
1. person
2. animals part 1
3. OOP for data science